crates/api_common/src/person.rs

```diff
@@ -422,3 +422,20 @@ pub struct UpdateTotp {
 pub struct UpdateTotpResponse {
   pub enabled: bool,
 }
+
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Get your user's image / media uploads.
+pub struct ListMedia {
+  pub page: Option<i64>,
+  pub limit: Option<i64>,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
I believe this can also derive `PartialEq`, `Eq`, and `Default` as well.

Reply:
Looking in this file, the request structs use a different set of derives than the response ones.